### PR TITLE
Skeleton structure for adding time-in-review

### PR DESCRIPTION
**Pull request by prince-chrismc**
towards #11

### CODE REVIEW

**Comment by prince-chrismc** [comment]
Looks like crap!

output

**Comment by prince-chrismc** [comment]
Improvement... but does it mean anything?

output

**Comment by prince-chrismc** [comment]
Somewhat helpful?

output

**Comment by prince-chrismc** [comment]
Looking better

output

**Comment by prince-chrismc** [comment]
@ericLemanissier @madebr 👋 I've been working on this, and I'd really love your thoughts.

If you get the chance please take a look at the [comment above](https://github.com/prince-chrismc/conan-center-index-pending-review/pull/12#issuecomment-815808514) the full data set can be seen [here](https://github.com/prince-chrismc/conan-center-index-pending-review/pull/12#issuecomment-815383623)

Thanks in advance!

**Comment by ericLemanissier** [comment]
quick question: what is the dotted red line ?

**Comment by prince-chrismc** [comment]
Moving average for the "days in review"

**Comment by prince-chrismc** [comment]
output

**Comment by prince-chrismc** [comment]
output

**Comment by ericLemanissier** [comment]
Nice work! The most usable line is probably the moving average one, so it would probably be better if it was the most readable. Maybe you could try to make it a solid line, and make the the others dotted ? Also, horizontal dotted reference line would help identifying long-term trends.

**Comment by prince-chrismc** [comment]
Missed posting https://github.com/prince-chrismc/conan-center-index-pending-review/pull/12/commits/e13d6da911e4991760bf9f9c3118a1f2603cf0b5 for my records

output

***

### Applying feedback

output

> make it a solid line, and make the the others dotted

I don't like how this rendered

output

**Comment by prince-chrismc** [comment]
> would help identifying long-term trends.

This is one of my goals but I've been struggling to pick a "sample size" for the moving average (bigger equals less sensitive --> better represents long trends) My dilemma is when we close 15-20 PRs, that day will greatly affect  

16 PR sample size

output

vs 30

output

vs 50

output

File: cmd/tir/time_in_review.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+			break
+		}
+		opt.Page = resp.NextPage
+	}
+
+	fmt.Println("::endgroup")
+
+	// if dryRun {
+	// 	return nil
+	// }
```
Comment (prince-chrismc) [inline cmd/tir/time_in_review.go:90]:
```suggestion
	if dryRun {
		return nil
	}
```

**Pull request merged**